hack/e2e/run.sh

```diff
@@ -267,6 +270,16 @@ fi
 
 OVERALL_TEST_PASSED="${TEST_PASSED}"
 
+if [[ "${METRICS}" == true ]]; then
+  metrics_collector "$KUBECONFIG" \
+    "$AWS_ACCOUNT_ID" \
+    "$AWS_REGION" \
+    "$NODE_OS_DISTRO" \
+    "$secondUsed" \
```

Comment:
I know this isn't your fault, but `$secondUsed` is such a bad name, can we change it to something like `$deployTimeSeconds` or whatever?